pkg/service/group_service_factory.go

```diff
@@ -67,64 +67,64 @@ type NsGroupService struct {
 	networkConfigService              *NetworkConfigService
 }
 
-// groupServiceOption :
-type GroupServiceOption struct {
-	Host         string
-	WaitOnJob    bool
-	Username     string
-	Password     string
-	isTenant     bool
-	ApiVersion   string
+// GroupServiceOptions :
+type GroupServiceOptions struct {
+	Host       string
+	WaitOnJob  bool
+	Username   string
+	Password   string
+	isTenant   bool
```

Comment:
Why lower case here and upper case for the others?  (Sorry I didn't notice this before.)

Reply:
I didn't realize this either. I'll make everything uppercase